cytosnake/utils/config_utils.py

```diff
@@ -118,3 +118,17 @@ def load_workflow_paths_config() -> dict:
     # load in _path.yaml and select key where all workflow paths are
     loaded_meta_paths = load_meta_path_configs()
     return loaded_meta_paths["workflow_dir"]["workflow"]
+
+
+def load_cytosnake_configs() -> dict:
+    """Loads in CytoSnake's general configuration
+
+    Returns
+    -------
+    dict
+        CytoSnake configs
+    """
+
+    # gets absolute path to cytosnake configs and load the configs
+    cytosnake_config_path = cp.get_cytosnake_config_path()
```

Comment:
I don't see where you define `cp`. It is not obvious to me

Reply:
Unfortunatly, you cannot see it within the changes (green), but if you extend the code view in reviewer mode, you will see at the top.
```python
import cytosnake.utils.cyto_paths as cp
```

Reply:
here's the [link](https://github.com/WayScience/CytoSnake/pull/45/files/596898faba88e29d1dfe5c95bec796ea920445e3#diff-68654fcf16b8e64472a858feacb4f03a540cbcbeb1cc065f8fc3cf10fd469f9bR5)

Reply:
ah! Gotcha. Is there a downside to using this functionality in this function given it is out of scope of the function definition?

Also, `cp` is a confusing acronym given it is also used for CellProfiler. Why not just use `cyto_paths`?

Reply:
That's a really good point. I just used `cp` because it was much easier to type. I did not take to account on how it can affect readability. 

Will change! easy fix